tests/ci/codebuild/linux-x86/run_accp_basic_tests.yml

```diff
@@ -0,0 +1,12 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - git submodule update --init --recursive
```

Comment:
Here and elsewhere: Can't CodeBuild be configured to do recursive clones by default and thus we don't need to execute this?

Reply:
we currently do that using [this Project configuration](https://docs.aws.amazon.com/codebuild/latest/APIReference/API_GitSubmodulesConfig.html) in the corretto account.

Reply:
Added the functionality in the cdk.